cpp/ql/src/semmle/code/cpp/models/interfaces/FormattingFunction.qll

```diff
@@ -7,6 +7,38 @@
  */
 
 import semmle.code.cpp.Function
+
+private Type stripTopLevelSpecifiersOnly(Type t) {
```

Comment:
Why was it necessary to introduce this? The commit that introduced it didn't come with any test output changes.

Reply:
It does have test changes, but in a test that's still internal (see internal PR https://git.semmle.com/Semmle/code/pull/28289).

Having said this, looking at it again I think I can do better.  I'll do this as follow-up work.